helper/install_devel_packages.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+source /etc/os-release
+
+case ${ID} in
+    fedora|centos|rhel)
+        dnf install \
```

Comment:
`dnf --setopt=install_weak_deps=False install` matches the `zypper` command, so you may want to do this here...

Reply:
ok done